src/layout/Header/ui/Header.tsx

```diff
@@ -0,0 +1,20 @@
+import { FC } from "react";
+import { useTranslation } from "shared/hooks/i18nHook";
+
+export const Header: FC = () => {
+
+    const { t } = useTranslation('main');
+
+    return (
+        <header className="w-full h-20 flex justify-between items-center">
+            <div>Logo</div>
+            <nav>
+                <ul>
+                    <li>
+                        <a href="/">{t('login')}</a>
```

Comment:
Let's also add button for switching languages here
<img width="397" alt="image" src="https://github.com/simeontg/cinema-frontend/assets/32066460/fcc4ba5f-9315-4130-b7e8-def74a26e541">